consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/statemachine/IbftRound.java

```diff
@@ -132,11 +138,15 @@ public void handleProposalMessage(final Proposal msg) {
 
     if (updateStateWithProposedBlock(msg)) {
       LOG.debug("Sending prepare message. round={}", roundState.getRoundIdentifier());
-      final Prepare localPrepareMessage =
-          messageFactory.createPrepare(getRoundIdentifier(), block.getHash());
-      transmitter.multicastPrepare(
-          localPrepareMessage.getRoundIdentifier(), localPrepareMessage.getDigest());
-      peerIsPrepared(localPrepareMessage);
+      try {
+        final Prepare localPrepareMessage =
+            messageFactory.createPrepare(getRoundIdentifier(), block.getHash());
+        peerIsPrepared(localPrepareMessage);
+        transmitter.multicastPrepare(
```

Comment:
you have changed the operations order from previous logic? i.e. Previously it was multiCastPrepare then peerIsPrepared. Now it is peerIsPrepared then multicastPrepare? Is that by mistake or it doesn't matter?

Reply:
that shouldn't matter in this instance.